_docs/_latest/configure/migrating-to-rego/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Migrating Forseti Scanners to Rego Constraints
+order: 800
+---
+
+# {{ page.title }}
+
+This guide will walk through migrating current Python scanner rule files to Rego 
+constraints found in the policy-library repository that can be used in Config Validator.
```

Comment:
Can you add some information about why some of the scanners do not have a migration guide? For example, CV only scans CAI exports right now and G Suite data is not available so there are no constraints/templates for the G Suite scanners. Also include info that for the scanners listed below, 100% of the scanner functionality is now available in the Policy Library templates.